homeassistant/components/songpal/media_player.py

```diff
@@ -132,6 +137,28 @@ async def async_will_remove_from_hass(self) -> None:
         """Run when entity will be removed from hass."""
         await self._dev.stop_listen_notifications()
 
+    async def _get_sound_modes_info(self):
+        """Get available sound modes and the active one."""
+        settings = await self._dev.get_sound_settings("soundField")
+        if isinstance(settings, Setting):
+            settings = [settings]
+
+        sound_modes = {}
+        active_sound_mode = None
+        for setting in settings:
+            cur = setting.currentValue
+            for opt in setting.candidate:
+                if opt.value == cur:
+                    active_sound_mode = opt.value
+                if not opt.isAvailable:
+                    continue
+                sound_modes[opt.value] = opt
+
+        _LOGGER.debug("Got sound modes: %s", sound_modes)
+        _LOGGER.debug("Active source: %s", active_sound_mode)
```

Comment:
Should this log really say "Active source" or should it be changed like this:
```suggestion
        _LOGGER.debug("Active sound mode: %s", active_sound_mode)
```